### ISSUE TITLE
New user registration fails in 4.2.3 when other content records on page

### ISSUE THREAD

**Issue by netcoop** [in2code-de/femanager#120]
Creating a new user with version 4.2.3 fails when the registration page contains more tt_content records then only the plugin record. Especially records with empty pi_flexform field return NULL instead of empty string for this field, causing an uncaught exception:

```
Uncaught TYPO3 Exception
Argument 2 passed to In2code\Femanager\Domain\Repository\PluginRepository::isViewInPluginConfiguration() must be of the type string, null given, called in .../Web/typo3conf/ext/femanager/Classes/Domain/Repository/PluginRepository.php on line 56
```

Solution: select only the fe_manager plugin records, and, just to avoid unnecessary exceptions, typecast the NULL value to string. I'll send a pull request.

**Comment by spoonerWeb** [comment]
I can confirm this. Breaks every registration if more than one CE on that page.

**Comment by einpraegsam** [comment]
I can confirm this issue and will search for a quick solution. Sorry for the trouble.

**Comment by einpraegsam** [comment]
Fix was commited in 4c97e6884320b31d7e2abbc98207b1faf7f3913c to develop branch. @sbusemann we should talk to each other if we should release a hotfix with just the regression updates. If so, develop branch was maybe not the right branch for this task (talking about git flow workflow).

**Comment by sbusemann** [comment]
resolved in 4.2.4 / 4.2.5
